package.json

```diff
@@ -14,6 +14,7 @@
     "babel-jest": "^26.3.0",
     "common-tags": "^1.8.0",
     "date-fns": "^2.17.0",
+    "directory-tree": "^3.0.0",
```

Comment:
Can we move this to `devDependencies` since I don't _think_ we need this part of the JS bundle for the site to function?

Reply:
o ye sure